test/v2/config/environment/testnet.json

```diff
@@ -1,6 +1,7 @@
 {
   "SRSPath": "../../../resources/srs",
   "KeyPath": "~/.test-v2/testnet-private-key.txt",
+  "KeyVar": "LIVE_TEST_TESTNET_HOLESKY_KEY",
```

Comment:
would be very useful to have a comment here as to which one has priority over the other one. Ive asked that we stop using json for contract configs and use toml everywhere. Think we should adopt this across the board. No need to do this as part of this PR but suggestion going forward.

yaml or toml both allow for comments, and both have parsers readily available in most languages.